documentation/record-reference.md

```diff
@@ -0,0 +1,109 @@
+# Record reference
+
+This document provides reference information about the record types,
+menus,
+link types
+and channel filters included with Base.
+
+## Record type definitions
+
+These sections describe common aspects of the record types:
+
+```{eval-rst}
+.. toctree::
+   :maxdepth: 1
+
+   generated/dbCommonRecord
+   generated/dbCommonInput
+   generated/dbCommonOutput
+```
+
+These are the record types supplied with EPICS Base:
+
+```{eval-rst}
+.. toctree::
+   :maxdepth: 1
+
+   generated/aaiRecord
+   generated/aaoRecord
+   generated/aiRecord
+   generated/aoRecord
```

Comment:
Another list that I would like to be generated if possible, plus the menu files below.

Reply:
This can of course be auto-generated, but would need some scripting to glue parts together. Markdown does not support including other files, at least trivially. I would be happy if auto-generation could be considered as a future enhancement.